utils/anautilities.py

```diff
@@ -556,7 +556,8 @@ def findInflectionPts(graph):
     # We are defining the inflection point as the point where the most negative gradient sum begins
     try:
         inflxGrad = negGrad[bigIdx][0]
-        inflxIdx = np.where(grad == inflxGrad) #return the index at the specified value
+        inflxIdices = np.where(grad == inflxGrad) #return the index at the specified value
+        inflxIdx = np.amax(inflxIdices) # if there are multiple possible points, take the largest one
```

Comment:
Is the variable `inflxIdices` necessary?

Reply:
Good point!